k8s/geth/faucet.deploy.yaml

```diff
@@ -1,103 +1,34 @@
         app: goerli
         version: v1
+        component: public-faucet
     spec:
-      priorityClassName: batch-priority
+      priorityClassName: production-priority
       containers:
-        - name: faucet
-          image: ethereum/client-go:alltools-stable
-          ports: 
-           - containerPort: 8080
-             name: api
-           - containerPort: 30303
-             name: discovery-tcp
-             protocol: TCP
-           - containerPort: 30303
-             name: discovery-udp
-             protocol: UDP
-          # Use /bin/sh -c to execute geth so that we have access to HOSTNAME in 
-          # the command arguments.
-          # https://github.com/kubernetes/kubernetes/issues/57726
-          command:
-            - "/bin/sh"
-            - "-c"
-            - > 
-              touch /tmp/pwd; # empty password file
-              faucet
-              -account.json=/data/accounts/account.json
-              -account.pass=/tmp/pwd
-              -apiport=8080
-              -bootnodes=enode://$(BOOTNODE_PUBKEY)@$(GETH_BOOTNODE_V5_SERVICE_HOST):$(GETH_BOOTNODE_V5_SERVICE_PORT_BOOTNODE_UDP)
-              -ethport=30303
-              -ethstats=$HOSTNAME:$(ETHSTATS_WS_SECRET)@$(GETH_ETHSTATS_SERVICE_HOST):$(GETH_ETHSTATS_SERVICE_PORT)
-              -faucet.amount=350
-              -faucet.minutes=1440
-              -faucet.name=validator-faucet
-              -faucet.tiers=3
-              -genesis=/data/genesis.json
-              -loglevel=3
-              -network=1337
-              -noauth
-          env:
-            - name: ETHSTATS_WS_SECRET
-              valueFrom:
-                secretKeyRef:
-                  name: ethstats-secrets
-                  key: ws
-            - name: BOOTNODE_PUBKEY
-              valueFrom:
-                secretKeyRef:
-                  name: geth-bootnode-secret
-                  key: public_key
-          volumeMounts:
-            - name: genesis
-              mountPath: "/data"
-              readOnly: true
-            - name: accounts
-              mountPath: "/data/accounts"
-              readOnly: true
-            - name: faucet-data
-              mountPath: "/root/.faucet/keys"
-          resources:
-            requests:
-              memory: "500Mi"
-              cpu: "50m"
-            limits:
-              memory: "750Mi"
-              cpu: "100m"
-      volumes:
-        - name: genesis
-          configMap: 
-            name: genesis
-            items:
-              - key: json
-                path: genesis.json
-        - name: accounts
-          secret:
-            secretName: geth-faucet-accounts-secret
-            items: 
-              - key: json
-                path: account.json
-        - name: faucet-data
-          emptyDir: {}
+      - name: faucet-grpc
+        image: gcr.io/prysmaticlabs/prysm/faucet:latest
+        args: 
+        - --recaptcha_secret=6LdX85EUAAAAAO-_qMEOdecIzxV3eH222ekzi1Y9
```

Comment:
This is a test/localhost secret. No need to be alarmed, but I shouldn't have committed it. I will remove it and rotate this key, later this week.

Reply:
We shouldn't merge this into master then,no ?

Reply:
It's not a big deal, helpful for localhost testing